Makefile

```diff
@@ -46,14 +46,17 @@ endif
 #----------------------------------------------------------------------
 
 # Explicitly set the default target to do everything that isn’t already done
-default: | init assets all public ;
+default: | init lint assets all public ;
 
 # Run anything that needs doing post-checkout to make this buildable
 init: node_modules ;
 
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
+	find source -type f -name '*.json' -print -exec jsonlint -q '{}' \;
```

Comment:
> It doesn't and even worse, it doesn't report the filename along with errors :'(

I'm thinking of a different linter. So yes, you're quite right to add a print so one might be able to puzzle out where it died in the event of failure.

> the problems with -exec aren't a big deal here

Ya I gave a :+1: for this changeset as is because it's pretty minor, more a note for future reference. The path error is actually something you should think about fixing in your RC files. Find is kind of anal about that, but it does have a point about security and working with project code you download off the internet it's something to think about.